tests/python/pants_test/build_graph/test_build_graph_integration.py

```diff
@@ -1,37 +0,0 @@
-# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
@stuhood @benjyw I'd appreciate a double check that it's fine to delete this IT, as we already test it through unit tests.

Cycles:
https://github.com/pantsbuild/pants/blob/124f04a44971d496f94587dc0605dee456e1a3d8/src/python/pants/engine/internals/build_files_test.py#L342-L358

Imports:
https://github.com/pantsbuild/pants/blob/124f04a44971d496f94587dc0605dee456e1a3d8/src/python/pants/engine/internals/build_files_test.py#L440-L472

Reply:
Not being sure about this was one of the reasons I left all this alone last week, so I'll defer to @stuhood , who already has cycles and such on the mind.

Reply:
But yeah, this seems fine to me!

Reply:
Yea, the other tests are much better. Thanks!